test/unit/utils-content.html

```diff
@@ -37,41 +37,46 @@
     <span></span>
   </x-content-multi>
 
+  <x-content-mixed id="elt4">
+    <span></span>
+    <x-content></x-content>
+  </x-content-mixed>
+
   <script>
-    
+
     suite('content utils', function() {
-      
+
       var elt1 = document.querySelector('#elt1');
       var elt2 = document.querySelector('#elt2');
       var elt3 = document.querySelector('#elt3');
-      
+
```

Comment:
Many trailing whitespaces died to bring us this PR...

![http://i.kinja-img.com/gawker-media/image/upload/s--MmWlPvc6--/195nhikg6hiunpng.png](http://i.kinja-img.com/gawker-media/image/upload/s--MmWlPvc6--/195nhikg6hiunpng.png)